Annotation/alternative_golden_set.json

```diff
@@ -974,13 +986,18 @@
             {
                 "predicate": [
                     {
-                        "str": "speak to",
+                        "str": "speak",
+                        "lemma": "",
+                        "POS": ""
+                    },
+                    {
+                        "str": "to",
                         "lemma": "",

```

Comment:
I'm not quite sure why you've structured it like this? considering we haven't been doing it like this thus far